app/views/profiles/edit.html.erb

```diff
@@ -1,12 +1,20 @@
 <h1><%= t('.title') %></h1>
 
-<header class="warning">
-  <p><%= t('.warning_html') %></p>
-</header>
+<%= form_with(model: user, url: profile_path) do |form| %>
```

Comment:
I'm fine merging this where it is, but I think it would be nice if this was located on the stories page for easier access. We can always address that later, though.